sdk/ml/azure-ai-ml/tests/workspace/e2etests/test_workspace.py

```diff
@@ -191,28 +195,132 @@ def test_uai_workspace_create_update_and_delete(
         assert workspace.description == wps_description
         assert workspace.display_name == wps_display_name
         assert workspace.public_network_access == PublicNetworkAccess.ENABLED
-        assert workspace.identity.type == ManagedServiceIdentityType.USER_ASSIGNED
+        assert workspace.identity.type == camel_to_snake(ManagedServiceIdentityType.USER_ASSIGNED)
         assert len(workspace.identity.user_assigned_identities) == 2
         assert workspace.primary_user_assigned_identity == user_assigned_identity.id
 
+        # test list
         workspace_list = client.workspaces.list()
         assert isinstance(workspace_list, ItemPaged)
 
+        # test get
         workspace = client.workspaces.get(name=wps_name)
+        assert isinstance(workspace, Workspace)
         assert workspace.name == wps_name
-        assert workspace.container_registry is None
-        assert workspace.identity.type == ManagedServiceIdentityType.USER_ASSIGNED
+        assert workspace.identity.type == camel_to_snake(ManagedServiceIdentityType.USER_ASSIGNED)
```

Comment:
Thanks for adding the tests. Please add a check for class type for Identity and user assigned identity.

Reply:
added, thanks for the advice.